server/src/main/java/com/objectcomputing/checkins/services/memberprofile/currentuser/CurrentUserDTO.java

```diff
@@ -1,132 +1,80 @@
 package com.objectcomputing.checkins.services.memberprofile.currentuser;
 
+import com.objectcomputing.checkins.services.memberprofile.MemberProfile;
 import io.micronaut.core.annotation.Introspected;
 import io.swagger.v3.oas.annotations.media.Schema;
 
 import javax.annotation.Nullable;
 import javax.validation.constraints.NotNull;
-import javax.validation.constraints.Past;
-import java.time.LocalDate;
-import java.util.UUID;
+import java.util.List;
+import java.util.Objects;
 
 @Introspected
 public class CurrentUserDTO {
```

Comment:
Wouldn't we need these? Like, if an existing user logs in to the system (with a completed member-profile), shouldn't we return the whole member-profile?

Reply:
We do. That is what the member profile is stored for. It is serialized and deserialized correctly. If you run the app and do a console log on user profile. You will be able to see the json returned back which includes the whole profile.